package.json

```diff
@@ -30,7 +30,7 @@
     "handlebars-helpers": "0.8.4",
     "handlebars-utils": "^1.0.6",
     "he": "^1.2.0",
-    "lodash": "^4.17.13",
+    "lodash": "^4.17.21",
```

Comment:
what is lodash used for in handlebars?

Reply:
It's an utility library, mostly used for iterating some different structures, objects.
I believe it's possisble to get rid of it

Reply:
Is that something you would advise doing if possible? Maybe a follow up if not simple to strip?

Reply:
I created a follow up ticket: https://jira.bigcommerce.com/browse/STRF-9673